Assets/Scripts/Stats.cs

```diff
@@ -15,7 +15,7 @@ public float currentHealth
             _currentHealth = value;
             if (value <= 0 && owner == EntityType.ENEMY) {
                 Destroy(gameObject);
-                int rand = Random.Range(0, 10);
+                int rand = Random.Range(0, 20);
```

Comment:
Would be great if we can make a custom method that input a percentage and gives us the desired logic rather than depending on Range. (again not need to do this now)